scripts/lib/spreadsheet-standardizer.ts

```diff
@@ -126,6 +157,35 @@ export class SpreadsheetStandardizer {
     }
     return authority in this.lowIncomeThresholds![state];
   }
+
+  createAmount(input: Record<string, string>): Record<string, number | string> {
+    const amount: Record<string, number | string> = {
+      type: input['amount.type'],
+      number: +cleanDollars(input['amount.number']),
+    };
+    if (
+      input['amount.representative'] !== undefined &&
+      input['amount.representative'] !== ''
+    ) {
+      amount.representative = +cleanDollars(input['amount.representative']);
+    }
+    if (
+      input['amount.minimum'] !== undefined &&
+      input['amount.minimum'] !== ''
+    ) {
+      amount.minimum = +cleanDollars(input['amount.minimum']);
+    }
+    if (
```

Comment:
Seeing this reused a lot - anyway to make into own function able to pass in the input type?

Reply:
I'm going to delete this method in a follow-up anyway once we have a collected data schema.

Reply:
Ack.